<html>
<head>
<title>BioMart</title>
</head>

<body>
<h1>BioMart</h1>

<p>BioMart is a simple, web-based tool for mining data from the WormBase ParaSite database.</p>

<p>There are two main parts to using BioMart, which can be navigated between in the menu on the left of the tool:</p>

<p><img src="/img/biomart_1.png" alt="BioMart" class="image-expand" style="width:400px;"></p>

<ul>
  <li><strong>Filters: </strong>defines the data you would like to include as input for your query</li>
  <li><strong>Attributes: </strong> defines the data types you want as output</li>
</ul>


<p><h3 id="toc1">1) Setting your Filters</h3></p>

<p>Filters are shown as a list in which each of the items can be expanded by clicking on the + sign:</p>

<p><img src="/img/biomart_2.png" alt="BioMart" class="image-expand" style="width:400px;"></p>

<p>The following filters are available:</p>

<ul>
  <li><strong>SPECIES:</strong> Use this filter to select either individual genomes or nematode clades. Multiple genomes can be selected by holding down the ctrl key or the option key on a Mac.</li>
  <li><strong>REGION:</strong> Restrict to a particular genomic region. Note that this filter should only be used where a single genome has been selected, as it is possibile that a particular region is present in multiple genomes.
  <p>If start/end co-ordinates are being specified, a scaffold or chromosome id is always required.</p>
  <p>Where multiple regions are specified, the format is 'Scaffold/Chr:Start:End:Strand' e.g. <a href="http://goo.gl/uusiH6">AG00032:411187:446321:1</a>. If no strand is specified, both strands are selected. Regions should be separated by a comma or new line.</p></li>
  <li><strong>GENE:</strong> Specify a list of genes with WormBase IDs, or one of the other ID types listed. IDs should be separated by a new line.</li>
  <li><strong>GENE ONTOLOGY:</strong> Restrict by one or more Gene Ontology (GO) terms for functional descriptions. Paste or upload a list of GO IDs or use the autocomplete box to populate the list. 
  <p>Alternatively, you can restrict to a particular GO evidence type e.g. Inferred by Electronic Annotation (IEA). For a explanation of GO evidence codes see the <a href="http://geneontology.org/page/guide-go-evidence-codes">GO website</a>. Multiple codes can be selected by holding down the ctrl key, or option key on a Mac.</p> </li>
  <li><strong>MULTI-SPECIES COMPARISONS: </strong>Restrict to orthologs or paralogs for a species by choosing from the pulldown list. You can choose to ONLY include that ortholog/paralog set or to EXCLUDE that set from your query by using the checkboxes. </li>
  <li><strong>PROTEIN DOMAINS:</strong> Allows you to restrict your query based on the presence or absence of protein domains. 
  <p><strong>Limit to genes...</strong>lets you choose a particular database feature set in include or exclude e.g. "restrict to all proteins containing any feature found in Pfam".</p>
  <p>The second option, <strong>Limit to genes with these family or domain IDs:</strong>, allows you to restrict to one or more protein domains/families. Accepts IDs from several databases including InterPRO, Pfam and Panther. IDs should be separated by a new line.</p></li>
</ul>

<p><h3 id="toc2">2) Setting your Attributes (outputs) </h3></p>

<p>By default, BioMart is set to return a two-column list with genome assembly ID and gene ID e.g.</p>

<p><img src="/img/biomart_3.png" alt="BioMart" class="image-expand" style="width:400px;"></p>

<p>However, the Attributes field can be further configured to return additional columns of data. These types can broadly be divided into four groups:</p>

<ol>
  
  <li><h4>Features:</h4></li>

<p><img src="/img/biomart_4.png" alt="BioMart" class="image-expand" style="width:400px;"></p>

<p>By default this option is set to return genome project information (e.g. ascaris_suum_prjna62057) and gene ID as two columns. </p>

<p><strong>Species</strong> information columns can also be included e.g. taxon ID and nematode clade. </p>

<p><strong>Gene attributes</strong> that can be selected to appear in the output include gene, protein and transcript IDs, scaffold IDs, gene and transcript starts and ends, gene and transcript names and descriptions.</p>

<p><strong>External references</strong> are cross-references to equivalent entities in other resources such as UniProt, RefSeq and ENA/GenBank as well as all WormBase IDs.</p>

<p><strong>GO and GOslim</strong> options output various GO data including name, GO ID, definition and evidence code and nearest ancestor 'GO slim' term (see the <a href="http://geneontology.org/page/go-slim-and-subset-guide">GO slim guide</a> for more information).</p>

<p><strong>Protein domain</strong> adds columns for protein attributes such as InterPro domains and descriptions.</p>
  
  <li><h4><strong>Structures:</strong></h4></li>
  
  <p><img src="/img/biomart_5.png" alt="BioMart" class="image-expand" style="width:400px;"></p>
  
  <p>In additional to Gene and Species attributes, this gives you output options for the exon structure of the gene such as CDS start and end and exon rank within the cDNA.</p>

  <li><h4><strong>Homologues:</strong></h4></li>
  
  <p><img src="/img/biomart_6.png" alt="BioMart" class="image-expand" style="width:400px;"></p>

<p>Gives the option of exporting orthologs or homologs from one or more chosen species for a set of input genes. For example "given me all <i>C. elegans</i> orthologs for the <i>B. malayi</i> genes eat-3, eat-4 and eat-6, eat-16 and eat-20".</p>

<p>As well as the core Gene and Species attributes, additional output options include ortholog gene and protein IDs, chromosome/scaffold ID, orthology confidence and % identity.</p>

  <li><h4><strong>Sequences:</strong></h4></li>    

<p>A very useful feature of BioMart is the ability to output sequence in many different configurations: </p>

<p><img src="/img/biomart_7.png" alt="BioMart" class="image-expand" style="width:400px;"></p>

<p>Output is a FASTA file where additional information can be given in the header lines. The sequences themsleves can be generated using the following configurations. Only one option can be selected at a time.</p>

<ul>
  <li><strong>Unspliced (Transcript)</strong></li>
  <p>Full transcript sequence, exons and introns. For genes with multiple transcripts, all transcript sequences are returned. </p>

  <li><strong>Unspliced (Gene)</strong></li>
  <p>Full gene sequence, exons and introns. </p>

  <li><strong>Flank (Transcript)</li></strong></li>
  <p>Sequence adjacent to a transcript. <strong>Upstream flank</strong> or <strong>Downstream flank</strong> needs to be specifed in the panel below, along with the length of the flanking sequence required in base pairs. For genes with multiple transcripts, all transcript flanking sequences are returned.</p>
  
 <li><strong>Flank (Gene)</strong></li>
 <p>Sequence adjacent to a gene. <strong>Upstream flank</strong> or <strong>Downstream flank</strong> needs to be specifed in the panel below, along with the length of the flanking sequence required in base pairs.</p>
 
 <li><strong>Flank-coding region (Transcript)</strong></li>
 <p>Flanking sequence for a transcript + 5' UTR sequence for upstream flank and 3' UTR sequence for downstream flank. <strong>Upstream flank</strong> or <strong>Downstream flank</strong> needs to be specifed in the panel below, along with the length of the flanking sequence required in base pairs. For genes with multiple transcripts, all transcript sequences are returned.</p>
 
 <li><strong>Flank-coding region (Gene)</strong></li>
 <p>Flanking sequence for a transcript + 5' UTR sequence for upstream flank and 3' UTR sequence for downstream flank. <strong>Upstream flank</strong> or <strong>Downstream flank</strong> needs to be specifed in the panel below, along with the length of the flanking sequence required in base pairs. For genes with multiple transcripts, all transcript sequences are returned.</p>
 
 <li><strong>5' UTR</strong></li>
 <p>Sequence of the 5' untranslated region. Note that this sequence is only retrievable if the 5' UTR has been annotated for the gene(s) in question.</p>
 
 <li><strong>3' UTR</strong></li>
 <p>Sequence of the 3' untranslated region. Note that this sequence is only retrievable if the 3' UTR has been annotated for the gene(s) in question.</p>
 
 <li><strong>Exon sequences</strong></li>
 <p>returns all exonic (coding) sequences. Each exon is returned as a separate sequence with its own header. Check 'Exon ID' in the Header section to get sequential numbering of exons within the gene/transcript.</p>

<li><strong>cDNA sequences</strong></li></strong></li>
<p>All coding sequence including 5' and 3' UTRs.</p>

<li><strong>Coding sequence</strong></li>
<p>All coding sequence excluding 5' and 3' UTRs.</p>

<li><strong>Peptide</strong></li>
<p>Protein translation of the coding sequences. For genes with multiple transcripts, all transcript peptide sequences are returned.</p>
 
 </ul>
 
<h4>Including additional information in sequence headers</h4>

<p>Additional information can be added to sequences headers under the <strong>HEADER INFORMATION</strong> section. These extra columns include various gene and transcript attributes. Useful attributes to include are 'Gene name' and 'Gene description', and 'Transcript stable ID' which will allow you to differentiate transcripts where multiple transcripts exist for the same gene.</p>

<p>In addition, where exon sequence is being returned, several options are available under the <strong>Exon Information </strong>section for adding information to the headers of individual exon sequences. A useful thing to include would be 'Exon ID' which numbers the exons sequentially within the gene or transcript.</p>

</ol>

<p><h3 id="toc3">3) Downloading your Output</h3></p>



</body>
</html>
