<html>
<head>
<title>BioMart</title>
</head>

<body>
<h1>BioMart</h1>

<p>BioMart is a simple, web-based tool for mining data from the WormBase ParaSite database.</p>

<p>There are two main aspects to using BioMart, which can be navigated to in the menu on the left of the tool:

[insert img highlighting location of menu]

<ul>
  <li><strong>Filters: </strong>defines the data you would like to include as input for your query</li>
  <li><strong>Attributes: </strong> defines the data types you want as output</li>
</ul>
</p>

<p><h3 id="toc1">1) Setting your Filters</h3></p>

<p>Filters are shown as a list in which each of the items can be expanded:</p>

[insert img of filters]

<p>The following filters are available:</p>

<ul>
  <li><strong>SPECIES:</strong> Use this filter to select either individual genomes or nematode clades. Multiple genomes can be selected by holding down the ctrl key or the option key on a Mac.</li>
  <li><strong>REGION:</strong> Restrict to a particular genomic region. Note that this filter should only be used where a single genome has been selected, as it is possibile that a particular region is present in multiple genomes.
  <p>If start/end co-ordinates are being specified, a scaffold or chromosome id is always required.</p>
  <p>Where multiple regions are specified, the format is 'Scaffold/Chr:Start:End:Strand' e.g. <a href="http://goo.gl/uusiH6">AG00032:411187:446321:1</a>. If no strand is specified, both strands are selected. Regions should be separated by a comma or new line.</p></li>
  <li><strong>GENE:</strong> Specify a list of genes with WormBase IDs, or one of the other ID types listed. IDs should be separated by a new line.</li>
  <li><strong>GENE ONTOLOGY:</strong> Restrict by one or more Gene Ontology (GO) terms for functional descriptions. Paste or upload a list of GO IDs or use the autocomplete box to populate the list. 
  <p>Alternatively, you can restrict to a particular GO evidence type e.g. Inferred by Electronic Annotation (IEA). For a explanation of GO evidence codes see the <a href="http://geneontology.org/page/guide-go-evidence-codes">GO website</a>. Multiple codes can be selected by holding down the ctrl key, or option key on a Mac.</p> </li>
  <li><strong>MULTI-SPECIES COMPARISONS: </strong>Restrict to orthologs or paralogs for a species by choosing from the pulldown list. You can choose to ONLY include that ortholog/paralog set or to EXCLUDE that set from your query by using the checkboxes. </li>
  <li><strong>PROTEIN DOMAINS:</strong> Allows you to restrict your query based on the presence or absence of protein domains. 
  <p><strong>Limit to genes...</strong>lets you choose a particular database feature set in include or exclude e.g. "restrict to all proteins containing any feature found in Pfam".</p>
  <p>The second option, <strong>Limit to genes with these family or domain IDs:</strong>, allows you to restrict to one or more protein domains/families. Accepts IDs from several databases including InterPRO, Pfam and Panther. IDs should be separated by a new line.</p></li>
</ul>

<p><h3 id="toc2">2) Setting your Attributes</h3></p>

<p>By default, BioMart is set to return a two-column list with genome assembly ID and gene ID e.g.</p>

[insert img of biomart output]

<p>However, the Attributes field can be further configured to return additional columns of data. These types can broadly be divided into four groups:</p>

<ul>
  <li><strong>Features:</strong> By default this option is set to return genome project information (e.g. ascaris_suum_prjna62057) and gene ID as two columns. 
  <p>Additional <strong>Species</strong> information columns can also be included e.g. taxon ID and nematode clade. </p>
  <p><strong>Gene attributes</strong> that can be selected to appear in the output include gene, protein and transcript IDs, scaffold IDs, gene and transcript starts and ends, gene and transcript names and descriptions.</p>
  <p><strong>External references</strong> are cross-references to equivalent entities in other resources such as UniProt, RefSeq and ENA/GenBank as well as all WormBase IDs.</p>
  <p><strong>GO and GOslim</strong> options output various GO data including name, GO ID, definition and evidence code and nearest ancestor 'GO slim' term (see the <a href="http://geneontology.org/page/go-slim-and-subset-guide">GO slim guide</a> for more information).</p>
  <p>Finally, </p></li>
  <li><strong>Structures:</strong></li>
  <li><strong>Homologues:</strong></li>
  <li><strong>Sequences:</strong></li>    
</ul>

</body>
</html>
