<html>
<head>
<title>BLAST</title>
</head>

<body>
  
<h2>Running BLAST searches</h2>

<ol>
  <li><a href="#toc1">Entering a query sequence</a></li>
  <li><a href="#toc2">Selecting the species</a></li>
  <li><a href="#toc3">Selecting the search tool</a></li>
  <li><a href="#toc4">Recent BLAST tickets</a></li>
  <li><a href="#toc5">Results</a></li>
  <li><a href="#toc6">Configuration options</a></li>
</ol>

<p>
  BLAST is a sequence similarity search tool that can be used for both DNA and proteins. The BLAST tool compares user-supplied sequences to sequences in the WormBase ParaSite database and returns the best hits.
  </p>

<p><h3 id="toc1">1) Entering a query sequence</h3></p>

<p>Paste in a sequence (the suggested format is FASTA) or upload a sequence as a file. Up to 30 sequences can be added. If inputting multiple species, make sure a header (for example a FASTA header) separates each one.</p>

<p><img src="/img/blast_input.png" alt="BLAST add sequence" class="image-expand" style="width:400px;"></p>

<p><h3 id="toc2">2) Selecting the species to search against</h3></p>

<p>
  You can choose to perform the BLAST search against all genomes in ParaSite, all Nematoda, all Platyhelminthes or a custom set of species. Selecting 'Custom species list' brings up a window where you can select your species within a taxonomic tree:
</p>

<p><img src="/img/blast_species_select.png" alt="BLAST select" class="image-expand" style="width:400px;"></p>

<p>DNA and protein databases available for similarity searches. When searching against the DNA database you can choose:</p>

<ul>

  <li>Genomic sequence:
    <p>Where repetitve and/or low complexity regions are NOT masked
    </p>
  </li>

  <li>Genomic sequence (hard masked):
  <p>Where genomic sequences have been run through the RepeatMasker program and repetitve and/or low complexity regions have been masked as Ns
  </p>
  </li>

  <li>cDNAs (transcripts/splice variant)</li>

</ul>

<p><h3 id="toc3">3) Selecting the search tool</h3></p>

<p>The following options are available, depending on the combination of query sequence and search database selected:</p>

<ul>
 
  <li>BLASTN: nucleotide sequences against nucleotide databases</li>
  <li>BLASTX: six frame translation of a nucleotide sequence against a peptide database</li>
  <li>TBLASTX: translated nucleotide sequences against a translated nucleotide database</li>
  <li>BLASTP: peptide sequences against peptide database</li>
  <li>TBLASTN: peptide sequences `gainst a six frame translated nucleotide database</li>

</ul>

<p>For BLAST searches, you can change the 'Search Sensitivity' from normal to the following:</p>

<ul>

  <li>Near match (to find closer matches- more stringent settings than 'normal')</li>
  <li>Short sequences (for short sequences like primers: BLASTN only)</li>
  <li>Distant homologies (to allow lower-scoring pairs to pass through)</li>
  
</ul>

<p>
  Specific parameters for these configurations can be found by expanding the <a href="#toc6">Configuration options</a>. Alternatively, change the configuration options to customise your own BLAST search.
  </p>

<p>
  You can give an optional description to this search in the Description field.
  </p>
  
<p>
  Once your parameters are set, click RUN to start the search.
</p>

<p><h3 id="toc4">4) Recent BLAST tickets</h3></p>

<p>
  The table lists tickets that are currently running or recently completed. You can customise the table by showing/hiding column.
</p>

<p>
  The progression of the ticket gets automatically refreshed every 10 seconds until fully completed.
  </p>

<p>The icons on the right can be used to download the results of a ticket, edit and resubmit the ticket, share the ticket via URL or delete the ticket.</p>

<p>
  Click the View results link to see the results.
  </p>

<p><img src="/img/blast_results.png" alt="BLAST results" class="image-expand" style="width:400px;"></p>

<p><h3 id="toc5">5) Results</h3></p>

<p>The results are displayed in two sections:</p>

<ul>
  <li>Job details:
    <p>
    Details of the job include job name, search type (e.g. BLASTN), date and time the job was submitted and configuration settings.
    </p>

    <p>
    Click on the title or (-) to collaspe the Job details section.
    </p>
  
  </li>

  <li>
    Results table:
    
    <p>
  The table lists all hits in order of high to low score (and E-value) and can be customised to show/hide columns. The results can be sorted by any parameters available in the table.
    </p>

  <p>Follow the links in the results table to get:</p>

<ul>

  <li>Genomic location: shows the BLAST hit on the Region in detail view in the Location tab of the Genome Browser. The BLAST hit will appear as a red bar along the genome. You may want to click on the red bar to view a summary of the search, including E-Value, %ID, etc.

<p><img src="/img/blast_region_in_detail.png" alt="BLAST" class="image-expand" style="width:400px;"></p>

  </li>
  
  <li>Sequence: shows the genomic sequence or query sequence</li>

  <li>Alignment: shows the BLAST alignment</li>

</ul>

</li>

</ul>



<p>Click on the title or (-) to collaspe the Results table.</p>


<p><h3 id="toc6">6) Configuration Options</h3></p>


<h4>General options</h4>

<ul>
<li>Maximum number of hits to report:</li>

<p>Number of database hits that are displayed. The actual number of alignments may be greater than this. It varies from 10 to 5000 to 100000 and the default is 100.</p>

<li>Culling limit</span></li>

<p>
  This option is available for BLAST searches only, not BLAT. It varies from 1 to 999 and the default is 5. The limit will delete a hit that in enveloped by at least this many higher scoring hits.
  </p>
  
<li>E-value threshold:</li>

<p>
  Threshold below which results are reported. It varies from 1e-200 to 1000 and the default value is 10.
  </p>

</ul>

<h4>Scoring options</h4>

<h4>Filtering and masking options</h4>

</body>
</html>
