<html>
<head>
<title>Example Queries</title>
<meta name="order" content="3" />
</head>

<body>
  
<h1>Example Queries</h1>

<ol>
<p>
  <li><a href="#serpin">I need to design primers for my serpin gene in <i>Tricuris muris</i> but I want to check my gene model is correct based on my RNA-Seq data. Can I do this in WormBase ParaSite?</a></li><br />
  <li><a href="#orthologs">I'd like to extract <i>C. elegans</i> orthologs for <i>Nippostrongylus</i> genes involved in a particular process.</a></span><br />
  <li><a href="#orthologs2">I have a list of genes from <i>Ascaris suum</i> and would like to know which ones have orthologs in humans and mammals and which ones might be nematode-specific:</a></li><br />
  <li><a href="#seq">I need the sequences for a set of <i>Schistosoma mansoni</i> genes. I have the chromosome, start, and stop for each:</a></li><br />
  <li><a href="#sigpep">I need a list of genes with predicted signal peptide that are present in <i>Brugia malayi</i>a given organism but not present in <i>C. elegans</i></a>
</p></li><br />
</ol>

<ol>

<li><h3 id="serpin">I need to design primers for my serpin gene in <i>Tricuris muris</i> but I want to check my gene model is correct based on my RNA-Seq data. Can I do this in WormBase ParaSite?</h3>

<ul>

<li>
  <p>Yes! First you will need to upload your RNASeq data into WormBase ParaSite.</p>

  <p>See the <a href="/info/Browsing/Upload/index.html#formats">Supported file formats</a> section for more information on what format   these files should be in.</p>

  <p>To add the data go to your region of interest for your species, <a href="http://parasite.wormbase.org/Trichuris_muris_prjeb126/Location/View?db=core;g=TMUE_s0016004100;r=scaffold16:252493-273266;t=TMUE_s0016004100;time=1439908569187.187">for example</a>.
  </p>

</li>

<li>In the left panel, click <strong>Add your data</strong>.</li>

<p><img src="/img/adding_data.png" alt="Adding data" class="image-expand" style="width:400px;"></p>

<li><p>In the <strong>Data format</strong> pulldown, choose the correct format, in this case <a href="/info/Browsing/Upload/wig.html">BigWig</a>, and add the URL to your data, e.g.:</p></li>

<li><p><a href="http://ngs.sanger.ac.uk/production/parasites/wormbase/RNASeq_alignments/trichuris_muris_prjeb126/ERS092077.bw">http://ngs.sanger.ac.uk/production/parasites/wormbase/RNASeq_alignments/trichuris_muris_prjeb126/ERS092077.bw</a></p></li>

<p><img src="/img/adding_data_3.png" alt="Adding data" class="image-expand" style="width:400px;"></p>

<li>Click <strong>Add data</strong>. Choose a colour for your track and click <strong>Save</strong>. You should now see your RNASeq data displayed as new tracks.</li>

<li><p>Add additional tracks using the same steps, or you can attach a group of tracks together if they are organised into a <a href="Browsing/Upload/adding_trackhubs.html">Track Hub</a>.</p>
  
<p>The data will remain attached in your browser unless it is removed via the <strong>Configure this page</strong> link. Tracks can be preserved between browsers/computers, and can also with collaborators by creating a <a href="/info/Getting_started/user_accounts.html">WormBase ParaSite account</a>.</p></li>

<li>If you notice corrections that need to be made to gene models in WormBase ParaSite, please <a href="/Help/Contact">contact us</a>.</li>

</ul>

</li>


<li><h3 id="orthologs">I'd like to extract <i>C. elegans</i> orthologs for <i>Nippostrongylus</i> genes involved in a particular process.</h3>

<ul>

<li>You can do that using <a href="/biomart/martview/">BioMart</a>. In the SPECIES menu select Nippostrongylus, then in the MULTI-SPECIES COMPARISONS menu select <strong>Orthologous C. elegans genes -> Only</strong>.
</li>

<p><img src="/img/examples_1.png" alt="Examples" class="image-expand" style="width:400px;"></p>

<li>You can further refine this list by function, process or location by choosing one or more categories from the GENE ONTOLOGY list. Start typing in the upper box and choose your terms of interest from the autocomplete, they will be added to the box beneath.</li>

<li>Click the <strong>Results</strong> button, top left, to see your results. By default a two-column file is returned that contains gene ID and Genome Project. To configure different options for the output, select <strong>Attributes</strong> in the left menu. See the <a href="/info/Tools/biomart.html#toc2">help documentation</a> for more information.
</li>

</ul>

</li>

<li><p><h3 id="orthologs2">I have a list of genes from <i>Ascaris suum</i> and would like to know which ones have orthologs in humans and mammals and which ones might be nematode-specific:</p></h3>

<p>
<table>
<tr><td>ASU_00012</td></tr>
<tr><td>ASU_00034</td></tr>
<tr><td>ASU_00089</td></tr>
<tr><td>ASU_00299</td></tr>
<tr><td>ASU_00345</td></tr>
<tr><td>ASU_00890</td></tr>
<tr><td>ASU_00998</td></tr>
<tr><td>ASU_01289</td></tr>
<tr><td>ASU_03421</td></tr>
</table>
</p>

<ul>

<li>You can do that using <a href="/biomart/martview/">BioMart</a>. In the GENE menu paste in your list, then in the MULTI-SPECIES COMPARISONS select <strong>Orthologous human genes -> Excluded</strong>.
  </li>

<p><img src="/img/examples_2.png" alt="Examples" class="image-expand" style="width:400px;"></p>

<li><p>You can also run this query against against mouse orthologs by selecting <strong>Orthologous mouse genes -> Excluded</strong>. The results are the same in this case.</p>
  </li>

<li>Click the <strong>Results</strong> button, top left, to see your results. By default a two-column file is returned that contains gene ID and Genome Project. To configure different options for the output, select <strong>Attributes</strong> in the left menu. See the <a href="/info/Tools/biomart.html#toc2">help documentation</a> for more information.
  </li>

  </ul>

</li>

<li><h3 id="seq">I need the sequences for a set of <i>Schistosoma mansoni</i> genes. I have the chromosome, start, and stop for each:</h3></li>

<p>
<table>
  <tr><td>Smp.Chr_3:5955668:5958652:1</td></tr>
  <tr><td>Smp.SC_0395:8998:9186:1</td></tr>
  <tr><td>Smp.Chr_1:32692191:32693861:1</td></tr>
  <tr><td>Smp.Chr_3:26054871:26055183:-1</td></tr>
</table>
</p>

<ul>
  
<li>From the SPECIES filter choose <i>Schistosoma mansoni</i>. Open the REGION section and enter the list of co-ordinates under 'Multiple regions'. The list can be separated by commas or new lines.
</li>

<p><img src="/img/examples_3.png" alt="Examples" class="image-expand" style="width:400px;"></p>

<li>Click the <strong>Attributes</strong> option from the left menu. Check the <strong>Sequences</strong> option, then in the SEQUENCES section choose <strong>Unspliced (genes)</strong>. Click the <strong>Results</strong> button, top left.</li>

<p><img src="/img/examples_4.png" alt="Examples" class="image-expand" style="width:400px;"></p>

</li>

</ul>

<li><h3 id="sigpep">I need a list of genes with predicted signal peptide that are present in <i>Brugia malayi</i>a given organism but not present in <i>C. elegans</i>.</h3></li>
</ol>

<ul>

<li>You can do that using <a href="/biomart/martview/">BioMart</a>. In the SPECIES section choose <i>Brugia malayi</i>, then in the MULTI-SPECIES COMPARISONS select <strong>Orthologous C. elegans genes -> Excluded</strong>.
  </li>
  
  <li>In the PROTEIN DOMAINS section check <strong>Limit to genes...</strong> and from the menu select <strong>with signal P protein features -> Only</strong></li>

<p><img src="/img/examples_5.png" alt="Examples" class="image-expand" style="width:400px;"></p>

<li>Click the <strong>Results</strong> button, top left, to see your results. By default a two-column file is returned that contains gene ID and Genome Project. To configure different options for the output, select <strong>Attributes</strong> in the left menu. See the <a href="/info/Tools/biomart.html#toc2">help documentation</a> for more information.
  </li>

  </ul>

</body>
</html>
