<html>
<head>
<title>Example Queries</title>
<meta name="order" content="3" />
</head>

<body>
  
<h1>Example Queries</h1>

<ol>

<li><span style="font-weight: bold">I need to design primers for my serpin gene in <i>Tricuris muris</i> but I want to check my gene model is correct based on my RNA-Seq data. Can I do this in WormBase ParaSite?</span>

<ul>

<li>
  <p>Yes! First you will need to upload your RNASeq data into WormBase ParaSite.</p>

  <p>See the <a href="/info/Browsing/Upload/index.html#formats">Supported file formats</a> section for more information on what format   these files should be in.</p>

  <p>To add the data go to your region of interest for your species, <a href="http://parasite.wormbase.org/Trichuris_muris_prjeb126/Location/View?db=core;g=TMUE_s0016004100;r=scaffold16:252493-273266;t=TMUE_s0016004100;time=1439908569187.187">for example</a>.
  </p>

</li>

<li>In the left panel, click <strong>Add your data</strong>.</li>

<p><img src="/img/adding_data.png" alt="Adding data" class="image-expand" style="width:400px;"></p>

<li><p>In the <strong>Data format</strong> pulldown, choose the correct format, in this case <a href="/info/Browsing/Upload/wig.html">BigWig</a>, and add the URL to your data, e.g.:</p></li>

<li><p><a href="http://ngs.sanger.ac.uk/production/parasites/wormbase/RNASeq_alignments/trichuris_muris_prjeb126/ERS092077.bw">http://ngs.sanger.ac.uk/production/parasites/wormbase/RNASeq_alignments/trichuris_muris_prjeb126/ERS092077.bw</a></p></li>

<p><img src="/img/adding_data_3.png" alt="Adding data" class="image-expand" style="width:400px;"></p>

<li>Click <strong>Add data</strong>. Choose a colour for your track and click <strong>Save</strong>. You should now see your RNASeq data displayed as new tracks.</li>

<li><p>Add additional tracks using the same steps, or you can attach a group of tracks together if they are organised into a <a href="Browsing/Upload/adding_trackhubs.html">Track Hub</a>.</p>
  
<p>The data will remain attached in your browser unless it is removed via the <strong>Configure this page</strong> link. Tracks can be preserved between browsers/computers, and can also with collaborators by creating a <a href="/info/Getting_started/user_accounts.html">WormBase ParaSite account</a>.</p></li>

<li>If you notice corrections that need to be made to gene models in WormBase ParaSite, please <a href="/Help/Contact">contact us</a>.</li>

</ul>

</li>


<li><span style="font-weight: bold">I'd like to extract <i>C. elegans</i> orthologs for <i>Nippostrongylus</i> genes involved in a particular process.</span>

<ul>

<li>You can do that using BioMart. In the SPECIES menu select Nippostrongylus, then in the MULTI-SPECIES COMPARISONS menu select <strong>Orthologous C. elegans genes -> Only</strong>.
</li>

<p><img src="/img/examples_1.png" alt="Examples" class="image-expand" style="width:400px;"></p>

<li>You can further refine this list by function, process or location by choosing one or more categories from the GENE ONTOLOGY list. Start typing in the upper box and choose your terms of interest from the autocomplete, they will be added to the box beneath.</li>

<li>Click the <strong>Results</strong> button, top left, to see your results. By default a two-column file is returned that contains gene ID and Genome Project. To configure different options for the output, select <strong>Attributes</strong> in the left menu. See the <a href="/info/Tools/biomart.html#toc2">help documentation</a> for more information.
</li>

</ul>

</li>

<li><p><span style="font-weight: bold">I have a list of genes from <i>Ascaris suum</i> and would like to know which ones have orthologs in humans and mammals and which ones might be nematode-specific:</p></span>

<table>
<tr><td>ASU_00012</td></tr>
<tr><td>ASU_00034</td></tr>
<tr><td>ASU_00089</td></tr>
<tr><td>ASU_00299</td></tr>
<tr><td>ASU_00345</td></tr>
<tr><td>ASU_00890</td></tr>
<tr><td>ASU_00998</td></tr>
<tr><td>ASU_01289</td></tr>
<tr><td>ASU_03421</td></tr>
</table>

<li>You can do that using BioMart. In the GENE menu paste in your list, then in the MULTI-SPECIES COMPARISONS select <strong>Orthologous human genes -> Excluded</strong>.



<p>You can also run this query against against mouse orthologs by selecting <strong>Orthologous mouse genes -> Excluded</strong>. The results are the same in this case.</p>
</li>

<li>Click the <strong>Results</strong> button, top left, to see your results. By default a two-column file is returned that contains gene ID and Genome Project. To configure different options for the output, select <strong>Attributes</strong> in the left menu. See the <a href="/info/Tools/biomart.html#toc2">help documentation</a> for more information.
</li>

</li>

<span style="font-weight: bold">I need the sequences for a set of Onchocerca volvulus genes. I have the chromosome, start, and stop for each:</span>

<li>Smp.Chr_3:5955668:5958652:1, Smp.SC_0395:8998:9186:1, Smp.Chr_1:32692191:32693861:1, Smp.Chr_3:26054871:26055183:-1

A: From the SPECIES filter choose S. mansoni, then under REGION enter the co-ordinates under 'Multiple regions'.

In 'Attributes' select 'Sequences' Unspliced (genes) and hit results.

</li>

</ol>

</body>
</html>
