<html>
<head>
<title>Example Queries</title>
<meta name="order" content="3" />
</head>

<body>
  
<h1>Example Queries</h1>

<ol>

<li><span style="font-weight: bold">I need to design primers for my serpin gene in <i>Tricuris muris</i> but I want to check my gene model is correct based on my RNA-Seq data. Can I do this in WormBase ParaSite?</span>

<ul>

<li>
  <p>Yes! First you will need to upload your RNASeq data into WormBase ParaSite.</p>

  <p>See the <a href="/info/Browsing/Upload/index.html#formats">Supported file formats</a> section for more information on what format   these files should be in.</p>

  <p>To add the data go to your region of interest for your species, <a href="http://parasite.wormbase.org/Trichuris_muris_prjeb126/Location/View?db=core;g=TMUE_s0016004100;r=scaffold16:252493-273266;t=TMUE_s0016004100;time=1439908569187.187">for example</a>.
  </p>

</li>

<li>In the left panel, click <strong>Add your data</strong>.</li>

<p><img src="/img/adding_data.png" alt="Adding data" class="image-expand" style="width:400px;"></p>

<li><p>In the ‘Data format’ pulldown, choose the correct format, in this case <a href="/info/Browsing/Upload/wig.html">BigWig</a>, and add the URL to your data, e.g.:</p></li>

<p><a href="http://ngs.sanger.ac.uk/production/parasites/wormbase/RNASeq_alignments/trichuris_muris_prjeb126/ERS092077.bw">http://ngs.sanger.ac.uk/production/parasites/wormbase/RNASeq_alignments/trichuris_muris_prjeb126/ERS092077.bw</a></p>

<p><img src="/img/adding_data_3.png" alt="Adding data" class="image-expand" style="width:400px;"></p>

<li><p>Click <strong>Add data</strong>.</p>

<p>Choose a colour for your track and click <strong>Save</strong>. You should now see your RNASeq data displayed as new tracks.</p></li>

<li><p>Add additional tracks using the same steps, or you can attach a group of tracks together if they are organised into a <a href="Browsing/Upload/adding_trackhubs.html">Track Hub</a>.</p>
  
<p>The data will remain attached in your browser unless it is removed via the <strong>Configure this page</strong> link. Tracks can be preserved between browsers/computers, and can also with collaborators by creating a <a href="/info/Getting_started/user_accounts.html">WormBase ParaSite account</a></p>.</li>

<li><p>If you notice corrections that need to be made to gene models in WormBase ParaSite, please <a href="/Help/Contact">contact us</a>.</p></li>

</ul>

</li>


<li><span style="font-weight: bold">I'd like to extract <i>C. elegans</i> orthologs for <i>Nippostrongylus</i> genes.</span>

<p>You can do that using BioMart.</p>

<p>In the SPECIES menu select Nippostrongylus, then in the MULTI-SPECIES COMPARISONS menu select 'Orthologous C. elegans genes' ONLY.</p>

<p>You can further refine this list by function or location by choosing one or more categories from the GENE ONTOLOGY list.</p>

</li>

Q: I have a list of genes from Ascaris suum and would like to know which ones have orthologs in human/mammals and which ones might be nematode specific:

ASU_00012
ASU_00034
ASU_00089
ASU_00299
ASU_00345
ASU_00890
ASU_00998
ASU_01289
ASU_03421

A: You can do that using BioMart. In the GENE menu paste in your list, then in the MULTI-SPECIES COMPARISONS select 'Orthologous human genes' EXCLUDED.

(Can also run against mouse orthologs - gives same result).

-- 

Q: I need the sequences for a set of Onchocerca volvulus genes. I have the chromosome, start, and stop for each:

Smp.Chr_3:5955668:5958652:1, Smp.SC_0395:8998:9186:1, Smp.Chr_1:32692191:32693861:1, Smp.Chr_3:26054871:26055183:-1

A: From the SPECIES filter choose S. mansoni, then under REGION enter the co-ordinates under 'Multiple regions'.

In 'Attributes' select 'Sequences' Unspliced (genes) and hit results.



</ol>

</body>
</html>
