<html>
<head>
<title>Frequently Asked Questions</title>
<meta name="order" content="4" />
</head>

<body>

<h1>Frequently Asked Questions (FAQs)</h1>

<p>
  <a href="#using">Using WormBase ParaSite</a></span><br />
  <a href="#datausage">Data Usage</a></span><br />
  <a href="#submission">Data Submission</a></span><br />
  <a href="#enquiries">General Enquiries</a>
</p> 

<h3 id="using">Using WormBase ParaSite</h3>
<p>
  <ol>
    <li><span style="font-weight: bold">How do I view my own data (e.g. RNA-Seq alignments) in the genome browser?</span><br />From within the genome browser, select the ‘Add your data’ or ‘Manage your data’ button in the left-hand menu.  In the popup panel, choose the genome which the data represents then select the format of your data file.  Depending on the data format chosen, you can either upload your file or enter the web address where it is hosted remotely.  Press ‘Attach’.  The genome browser will now refresh and your data will be shown as a track within the browser image.  Tracks can be switched on and off through the ‘Configure this page’ interface.</li>
    <li><span style="font-weight: bold">I have a list of genes from one species, how do I get the orthologues from another species?</span><br />Use our <a href="/biomart/martview">BioMart</a> tool. On the 'Filters' screen, choose your species of interest from the 'Species' section then add your list of gene names to the 'ID list limit' section under 'Gene'. On the 'Attributes' page, select the 'Homologues' radio button, then expand the 'Orthologues' section. Choose the relevant data you would like to get for the target species. Click 'Results' to see the full list.</li>
    <li><span style="font-weight: bold">I have a list of genes from one species and would like to know which of these do not have a human orthologue.</span><br />Use our <a href="/biomart/martview">BioMart</a> tool. On the 'Filters' screen, choose your species of interest from the 'Species' section then add your list of gene names to the 'ID list limit' section under 'Gene'. In the 'Multi-species Comparisons' section, select 'Orthologous Human Genes excluded'. Click 'Results' to see the gene list.</li>
    <li><span style="font-weight: bold">Can I access WormBase ParaSite data programmatically without downloading large files?</span><br />We offer a <a href="/api">comprehensive REST API</a> which allows data from WormBase ParaSite to be accessed by scripts. Should you wish to embed a script containing calls to our REST API in a public website, please contact us first, so we can ensure access for other users is not disrupted.</li>
    <li><span style="font-weight: bold">Why is Clade II missing from the nematoda section of the homepage and not included as a clade in the BLAST tool?</span><br />There are no parasitic clade II nematodes.</li>
    <li><span style="font-weight: bold">How do I BLAST just the coding sequence of my gene?</span><br />Go to the corresponding transcript page for your gene by clicking on the Transcript tab at the top of the window. To BLAST individual exons go to Sequence > Exons in the left menu. Select your exon of interest and click 'BLAST this sequence' from the popup menu. To BLAST all coding sequence, choose Sequence > cDNA from the menu.</li>
  </ol>
</p>

<h3 id="datausage">Data Usage</h3>
<p>
  <ol>
    <li><span style="font-weight: bold">How should I cite genomes taken from WormBase ParaSite?</span><br />You should cite the original paper, which is linked to from the relevant species information page. If the genome is unpublished, you are expected to comply with our <a href="/info/about/datausage.html">data usage policy</a>, which requires contacting the principal investigator before undertaking large scale projects.  Additionally, you are asked to cite WormBase ParaSite, details of the relevant publication(s) are outlined in our <a href="/info/about/datausage.html">data usage policy</a>.</li>
  </ol>
</p>

<h3 id="submission">Data Submission & Updates</h3>
<p>
  <ol>
    <li><span style="font-weight: bold">Why do you not display the genome for a particular species?</span><br />If you are aware of a genome assembly and set of annotated gene models that are not included in WormBase ParaSite, please <a href="/Help/Contact">contact us by email</a> and we will consider including the genome.</li>
    <li><span style="font-weight: bold">How do I submit my data to WormBase ParaSite?</span><br />WormBase ParaSite is happy to display new or recommended genomic and transcriptomic data where possible. As we are not a data repository we ask that your sequences are first deposited in a nucleotide archive such as ENA or GenBank then please <a href="/Help/Contact">contact us by email</a> with details.</li>
    <li><span style="font-weight: bold">I would like to propose a change to the description for a species, how do I do this?</span><br />Please <a href="/Help/Contact">contact us by email</a> with your proposed changes to the description.</li>
    <li><span style="font-weight: bold">I’ve noticed a mistake in a gene model for one of the species in WormBase ParaSite. Can you fix it?</span><br />We are happy to fix gene annotation for ‘core’ species maintained by WormBase. These are listed on the WormBase website. For non-core species, please contact the authors of the genome assembly.</li>
  </ol>
</p>

<h3 id="enquiries">General Enquiries</h3>
<p>
  <ol>
    <li><span style="font-weight: bold">Can you provide medical advice regarding a parasitic worm infection?</span><br />WormBase is a scientific database and cannot provide medical information or help.  If you need such advice, please consult a qualified physician.</li>
    <li><span style="font-weight: bold">I have a question that is not listed here</span><br />Feel free to <a href="/Help/Contact">contact us by email</a>.</li>
  </ol>
</p>
</body>
</html>
