<html>
<head>
<meta name="order" content="1" />
<title>Large File Formats</title>
</head>

<body>

<h1>Large File Formats</h1>

<p>WormBase ParaSite supports a number of large (multi-gigabyte) file formats for datasets that are too large to upload directly to the our servers. The file remains on the remote server and is accessed via its URL, with only small chunks of data being requested by the webcode at any one time.</p>

<h2 id="bam-format">BAM format</h2>
<p>BAM is a compressed version of the SAM (Sequence Alignment/Map) binary format. BAM uses an index file to give fast access to small sections of the file.</p>
<p>Please note that if you are attaching a BAM file from a URL, you will need to have an index file with the extension .bam.bai in the same directory as your data file, with the same name.</p>
<p>Additional information about SAM/BAM is available at the <a href="http://samtools.sourceforge.net/">SAMtools development site</a>.

<h2 id="bb-format">bigBED format</h2>

<p>BigBED is an indexed format created from standard <a href="/info/website/upload/bed.html">BED</a> files using the <kbd>bedToBigBed</kbd> utility program. It allows large datasets to be processed much faster than a conventional BED file.</p> 

<h2 id="bw-format">BigWig format</h2>

<p>The BigWig format is designed for dense, continuous data that is intended to be displayed as a graph. Files can be created from WIG or BedGraph files using the appropriate utility program.</p>

<p>WormBase ParaSite currently allows the following configuration options to be set when you upload the file:</p>

<ul>
<li>Track colour (10 choices)</li>
</ul>

<h2 id="vcf-format">VCF format</h2>

<p>The VCF format is a tab delimited format for storing variant calls and and individual genotypes. It is able to store all variant calls from single nucleotide variants to large scale insertions and deletions.</p>

<p>More information on this format can be obtained from the <a href="http://samtools.github.io/hts-specs/">SAMtools GitHub site</a>.</p>

<p>Please note that if you are attaching a large VCF file from a URL, you will need to have an index file with the extension .vcf.gz.tbi in the same directory as your data file, with the same name.</p>

<p>In order to produce the indexed vcf file with the .gz.tbi extension you must follow the following steps:</p>
<ul>
<li>Compress your vcf file using <a href="http://sourceforge.net/projects/bgzip/">bgzip</a></li>
<li>Index the vcf.gz file using <a href="http://samtools.sourceforge.net/tabix.shtml">tabix</a>. Use will need to pass the option -p vcf to tabix, for example "/usr/bin/tabix -p vcf my_file.vcf.gz"</li>
</ul>

</body>
</html>
