<html>
<head>
<meta name="order" content="1" />
<title>Large File Formats</title>
</head>

<body>

<h1>Large File Formats</h1>

<p>WormBase ParaSite supports a number of large (multi-gigabyte) file formats for datasets that are too large to upload directly to the our servers. The file remains on the remote server and is accessed via its URL, with only small chunks of data being requested by the webcode at any one time.</p>

<h2 id="bam-format">BAM format</h2>
<p>BAM is a compressed version of the SAM (Sequence Alignment/Map) binary format. BAM uses an index file to give fast access to small sections of the file.</p>
<p>Please note that if you are attaching a BAM file from a URL, you will need to have an index file with the extension .bam.bai in the same directory as your data file, with the same name.</p>
<p>Additional information about SAM/BAM is available at the <a href="http://samtools.sourceforge.net/">SAMtools development site</a>.

<h2 id="bb-format">BigBED format</h2>

<p>BigBED is an indexed format created from standard <a href="bed.html">BED</a> files using the <kbd>bedToBigBed</kbd> utility program which is part of the <a href="https://github.com/arq5x/bedtools2">bedtools toolset</a>. It allows large datasets to be processed much faster than a conventional BED file.</p> 

<h2 id="bw-format">BigWig format</h2>

<p>The BigWig format is designed for dense, continuous data that is intended to be displayed as a graph. Files can be created from WIG or BedGraph files using the appropriate utility program.</p>

<h2 id="vcf-format">VCF format</h2>

<p>The VCF format is a tab delimited format for storing variant calls and and individual genotypes. It is able to store all variant calls from single nucleotide variants to large scale insertions and deletions.</p>

<p>More information on this format can be obtained from the <a href="http://samtools.github.io/hts-specs/">SAMtools specifications site</a>.</p>

<p>Please note that if you are attaching a large VCF file from a URL, you will need to have an index file with the extension .vcf.gz.tbi in the same directory as your data file, with the same name.</p>

<p>In order to produce the indexed vcf file with the .gz.tbi extension you must follow the following steps:</p>
<ul>
<li>Compress your vcf file using <a href="http://sourceforge.net/projects/bgzip/">bgzip</a></li>
<li>Index the vcf.gz file using <a href="http://samtools.sourceforge.net/tabix.shtml">tabix</a>. Use will need to pass the option -p vcf to tabix, for example "/usr/bin/tabix -p vcf my_file.vcf.gz"</li>
</ul>

<h2 id="vep-format">VEP format</h2>

<p>VEP (Variant Effect Predictor) format is a simple <strong>whitespace-separated</strong> format (columns may be separated by space or tab characters), containing five required columns plus an optional identifier column:
</p>
    
    <ol>
        <li><strong>chromosome</strong> - just the name or number, with no 'chr'
        prefix</li>
        <li><strong>start</strong></li>
        <li><strong>end</strong></li>
        <li><strong>allele</strong> - pair of alleles separated by a '/', with the
        reference allele first</li>
        <li><strong>strand</strong> - defined as + (forward) or - (reverse).</li>
        <li><strong>identifier</strong> - this identifier will be used in the VEP's
        output. If not provided, the VEP will construct an identifier from the given
        coordinates and alleles.</li>
    </ol>
    
    <pre class="code">
1   881907    881906    -/C   +
5   140532    140532    T/C   +
12  1017956   1017956   T/A   +
2   946507    946507    G/C   +
14  19584687  19584687  C/T   -
19  66520     66520     G/A   +    var1
8   150029    150029    A/T   +    var2</pre>
    
    <p>An insertion (of any size) is indicated by start coordinate = end coordinate
    + 1. For example, an insertion of 'C' between nucleotides 12600 and 12601 on the
    forward strand of chromosome 8 is indicated as follows:</p>
    
    <pre class="code">
8   12601     12600     -/C   +</pre>
    <p>A deletion is indicated by the exact nucleotide coordinates. For example, a
    three base pair deletion of nucleotides 12600, 12601, and 12602 of the reverse
    strand of chromosome 8 will be:</p>
    
    <pre class="code">
8   12600     12602     CGT/- -</pre>


    

</body>
</html>
