<html>
<head>
<title>Genome Statistics Widget</title>
</head>

<body>

  <h1>Genome Statistics</h1>

  <img src="/img/assembly_widget.png" alt="Assembly statistics widget" class="image-expand" />
  
  <h2>Genome General Information</h2>
  <p>Some general information and statistics are displayed for the assembly/annotation of the genome.</p> 

  <h2>Scaffold Statistics Widget</h2>
  <p>The circumference of the widget represents the full length of the genome, from 0% to 100%, with the scaffolds sorted into descending order based on length.  The radius of the silver area represents the scaffold length at this point in the descending list.  The length of the longest scaffold within the assembly is shown in orange.</p>
  <p>Dark green = N50 length.  This is length of the smallest contig such as the sum of the sequences larger than this contig covers half of the genome assembly.</p>
  <p>Light green = N90 length.  This is length of the smallest contig such as the sum of the sequences larger than this contig covering 90% of the genome assembly.</p>
  <p>Hovering over the widget will reveal the N-value at this point.  For example, hovering over 30% will display the N30 value.</p>
  <p>The exterior of the circle shows the AT:GC:N content ratio within each scaffold at that point of the genome.</p>
  <h2>BUSCO ANNOTATION Piechart</h2>
  <p>BUSCO ANNOTATION (--mode proteins) is running at the protein level, assessing not only the assembly quality of a genome but also the quality of its annotation. In the genome annotation, we look for single-copy orthologues that are present in more than 90% of animals. The percentages of complete (duplicated and single) and fragmented genes recovered are reported.</p>
  <h2>BUSCO ASSEMBLY Piechart</h2>
  <p>BUSCO ASSEMBLY (--mode genome) does not take account the genome's provided gene models, assessing the assembly quality of a genome. It initially runs AUGUSTUS on the genome's assembly, to predict a gene-set. In the AUGUSTUS predicted gene-set for the genome assembly, we look for single-copy orthologues that are present in more than 90% of animals. The percentages of complete (duplicated and single) and fragmented genes recovered are reported.</p> 

  <h2>BUSCO Metrics Calculation</h2>
  <p><a href="https://busco-archive.ezlab.org/">BUSCO v3</a> is used to compute these metrics.</p>
  <p>Additionally, we use a different database for assessing nematodes and platyhelminths: Nematode assemblies use nematoda_odb9 for BUSCO analysis, Platyhelminth assemblies use metazoa_odb9 for BUSCO analysis.</p>

</body>
</html>
