<html>
<head>
<title>Gene Pages</title>
</head>

<body>

<h2>Gene Pages</h2>

<p>The gene page is the source of information relating to a gene. All gene pages have a header section which summarizes the key information about the gene:</p>

<p><img src="/img/gene_header_section.png" alt="Gene page header" class="image-expand" style="height:150px;"></p>

<p>This includes (from top to bottom): Gene name and gene id, a short gene description (where available), location of the gene, additional information about the gene including number of orthologs, gene type, annotation method and a table of all alternative transcripts of the gene.</p>

<p>The menu on the left-hand side allows you navigate through available data:</p>

<p><img src="/img/gene_page_1.png" alt="Gene page" class="image-expand" style="height:400px;"></p>

<p>Available data are:</p>

  <h3>Summary</h3>
  This is a summary view of the gene model, and any of its neighbours, on the genome. For a more interactive view, follow the <a href="region_in_detail.html">Region in detail</a> link.

  <h3>Splice variants</h3>
  Displays all known splice variants as a table and in a browser view.

  <h3>Sequence</h3>
  This displays the nucleotide sequence of the gene. Exons are highlighted in pink/red:
  <p><img src="/img/gene_page_2.png" alt="Gene page" class="image-expand" style="height:300px;"></p>
  This page also gives you the option of downloading the sequence in FASTA or RTF formats, or to BLAST the sequence using the WormBase ParaSite <a href="Tools/blast.html">BLAST service</a>.

  <h3>External references</h3>
  Links to external databases such as EntrezGene and WormBase.

  <h3>Ontology</h3>
  Displays all available Gene Ontology annotation data from the three ontologies: biological process, molecular function and cellular component. Clicking on the 'Chart' icon for a particular term displays a tree view of for that GO term up to the root node, and selecting the Ancestry chart tab shows a tree view for all annotated terms in the selected ontology.

  <h3>Comparative Genomics</h3>
    <ul>
      
      <li>Gene tree</li>
      Comparative genomics tree for the gene generated using the <a href="http://www.ensembl.org/info/genome/compara/index.html">Ensembl Compara tools</a>.
      
        <p><img src="/img/gene_page_3.png" alt="Gene tree" class="image-expand" style="height:300px;"></p>
        
      <p>The alignments on the right of the gene tree shows areas of synteny between the sequences.</p>
      
      <p>The tree can be filtered using the table above by GO term or InterPro domain. For example: display a tree for only those genes containing InterPro domain IPR000156 Ran binding domain.</p>
      
      <p>Branches of the tree can be expanded and collapsed by clicking the nodes. The image can be configured and downloaded using the icons at the top left.</p>
      
      <li>Orthologues</li>
      Displays a list of all orthologous genes (i.e. evolved from a common ancestral gene) in species in WormBase ParaSite as well as some other comparitor and model species including Caenorhabiditis elegans, human, Drosophila melanogaster,  Mouse and Zebrafish. The list can be filtered using the summary table at the top - by default all orthologs are displayed.
      
      <p>The alignment data underlying the table can be downloaded in a variety of different formats including FASTA, CLUSTAL W, Mega, MSF, Nexus, OrthoXML, Pfam, Phylip, PSI and Stockholm by following the link at the top of the table.</p>
      
      <li>Paralogues</li>
      Displays a list of all paralogous genes (i.e. homologous gene sequences created by a duplication events within a genome). 
      
        <p>The alignment data underlying the table can be downloaded in a variety of different formats including FASTA, CLUSTAL W, Mega, MSF, Nexus, OrthoXML, Pfam, Phylip, PSI and Stockholm by following the link at the top of the table.</p>
        
    </ul>

</body>
</html>
